lib/config.js

```diff
@@ -548,3 +550,15 @@ function loadConfigFile (configFile) {
 
   return null
 }
+
+function loadServerCaCertFile (opts) {
+  if (opts.serverCaCertFile) {
+    try {
+      return fs.readFileSync(opts.serverCaCertFile)
+    } catch (err) {
+      opts.logger.error('Elastic APM initialization error: Can\'t read server CA cert file %s', opts.serverCaCertFile)
+      opts.logger.error('APM Server transport error:', err.stack)
+    }
+  }
+  return undefined
```

Comment:
Not important, but just FYI. In JavaScript, a `return undefined` is implied in functions if either no `return` is called, or `return` without any value is called - so we normally don't explicitly do this.

Reply:
thanks!